docs/src/explanations/interfaces-and-connections.md

```diff
@@ -67,6 +68,31 @@ class CrossbarIo(n: Int) extends Bundle {
 where Vec takes a size as the first argument and a block returning a port as the second argument.
 
 ## Bulk Connections
+Once we have a defined Interface, we can connect to it via a monoConnect operator (:=) or biconnect operator (<>)
+### Monoconnect Algorithm
+MonoConnect.connect executes a mono-directional connection element-wise.
+
+Note that this isn't commutative. There is an explicit source and sink
+already determined before this function is called.
+
+The connect operation will recurse down the left Data (with the right Data).
+An exception will be thrown if a movement through the left cannot be matched
+in the right. The right side is allowed to have extra Record fields.
+Vecs must still be exactly the same size.
+
+See elemConnect for details on how the root connections are issued.
+
```

Comment:
```suggestion
```

this is now describing code that's actually in the MonoConnect.scala file. I'm not sure we need this level of detail here